avatars/client_test.py

```diff
@@ -50,3 +39,44 @@ def test_url_is_rejected_if_it_contains_quotes(base_url: str) -> None:
     # env variable configuration.
     with pytest.raises(ValueError, match="not to contain quotes") as exc_info:
         api_client = ApiClient(base_url=base_url)
+
+
+import httpx
+
+RequestHandle = Callable[[httpx.Request], httpx.Response]
+
+
+def api_client_factory(handler: Optional[RequestHandle] = None) -> ApiClient:
```

Comment:
Why do you have 2 `api_client_factory`?

Reply:
All of this is a wrong copy paste no?
(import in the wrong place too)

Reply:
yes